### PR TITLE
LA-11831 Cosmos nosql perms check script.

### PR DESCRIPTION
**Pull request by kaif-lb**

### CODE REVIEW

File: sql_user_check_cosmos_nosql/run.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly CMD="az"
+readonly RESOURCE_GROUP=$RESOURCE_GROUP
+readonly ACCOUNT_NAME=$ACCOUNT_NAME
+readonly CLIENT_ID=$CLIENT_ID
```
Comment (kasim-lb) [inline sql_user_check_cosmos_nosql/run.sh:8]: Any reason we have bash script ?
Instead can we have python script inside servicbility pod ?

**Review by kasim-lb — CHANGES_REQUESTED** [review]
LGTM

**Review by kasim-lb — APPROVED** [review]
Check with @arpit-lb , if we can move this script to serviceability pod

File: sql_user_check_cosmos_nosql/run.sh (hunk 1 above)
Comment (kaif-lb) [reply sql_user_check_cosmos_nosql/run.sh:8]: This repo is public, when customers prepare credentials, we ask them to run script from this repo to validate credentials before onboarding. All the scripts here are in bash.

**Review by atreyee-lb — APPROVED** [review]

**Pull request merged**